bn-apps/ledger-sync/ledger-sync-service/src/main/kotlin/com/r3/businessnetworks/ledgersync/Helpers.kt

```diff
@@ -73,6 +72,9 @@ fun List<SecureHash>.hash(): SecureHash = map {
  */
 @CordaService
 class ConfigurationService(appServiceHub : AppServiceHub): SingletonSerializeAsToken()  {
```

Comment:
`appServiceHub` is unused.

Reply:
it is actually required by SingletonSerializeAsToken. I actually tried to remove it before, and it caused error.

Reply:
optimize import will do